notebooks/Liquidators_data.ipynb

```diff
@@ -0,0 +1,1052 @@
3iiYmJcHJyAgB4eXmhWrVqSE5OxrVr1+Q2VapUQZUqVXDq1CnodDo5HhISAm9vbxw5cgTZ2dlyvFatWnB1dUViYqLJm7NBgwawt7fHwYMHTXJo2rQpcnNzcfjwYTmmVCrRrFkz6HQ6nDhxQo5rNBqEhYXh+vXrOHv2rBzXarWoXbs2Ll++bDL4iX1in9gn9ol9Yp+s7pP6nYI+5UxDruSCw/bD7/YJuWiWMw06RRBO2PW72ydxHWG5c3Bd2QBnVd3u9slwFrXzluOyshUuqZ652yd9Eqrlb0CyKgLXlA3v9il/N6rod+GUXR/oFCF3+5S/Ad76JByxH4RsyfNun/KWw9VwFonqUXcLSAANcufAXmTg4L99wb99K+/t5OXlBVtg0TWVlvjrr7/QuHHjB/55xqLXVJ49exbVqlXDn3/+iUaNGsntevXqBVdXVyxcuBBxcXFo3749bt68aXK0MiwsDL1798bEiRMxf/58vP3222ajvV1dXTFz5kwMHDgQn3zyCX777Tf89ddf8vS0tDS4u7sjLi6uxNPfxR2pDAgIwI0bN+RrHvjtln1in9gn9ol9Yp8A/FxwsOWxOlLZN6v4vhr7VEbbKTMz8/G5pvJRCA4Ohq+vL7Zu3SoXlbm5udi5cye++OILAECTJk1gZ2eHrVu3om/fvgCAlJQUHDlyBF9++SUAIDw8HDqdDvv375dvzr5v3z7odDq0bNlSbjN58mSkpKTAz88PQMFvnKvVajRp0qTEHNVqNdRqtVlcpVJBpTJdtcYPY1HGN6Kl8aLzfZC4JEnFxkvK0do4+8Q+lRRnn9gngH0qKUdr47bXp8K36RNQwfy2fRIMxcYLisXi4gXFonk8v9jb2xQUhZbHi8vFJF6kz+W5nWzBIy0qMzMz8X//93/y8+TkZCQlJcHd3R1Vq1bF6NGjMWXKFNSoUQM1atTAlClT4OjoiP79+wMoOFQ8ePBgjBkzBh4eHnB3d8fYsWNRv359dOjQAQBQu3ZtdO7cGUOHDsWcOXMAAMOGDUP37t0RGhoKAOjUqRPq1KmDyMhITJs2DTdv3sTYsWMxdOjQCv2NgIiIiKiieKRF5cGDB01OLb/99tsAgAEDBiA6OhrvvvsusrOzMWLECKSlpaF58+bYsmULnJ2d5dfMnDkTKpUKffv2RXZ2Ntq3b4/o6GiTb1JLly7FyJEj5VHiPXv2NLk3plKpxIYNGzBixAi0atUKGo0G/fv3x/Tp08t7FRARERE9Fiy+prJRo0b3HLRy+/ZtnD59+oGvqXwc2Mp9pIiIiB463qfygdlKfWHxkUrjABoiIiIioqIsLirHjx9fnnkQERERkQ2zzeFFRERERFShsKgkIiIiolJjUUlEREREpcaikoiIiIhKjUUlEREREZWaRaO/v/32W4tnOHLkyAdOhoiIiIhsk0VF5cyZMy2amSRJLCqJiIiInkAWFZXJycnlnQcRERER2TBeU0lEREREpWbxL+oUdunSJaxduxYXLlxAbm6uybQZM2aUSWJEREREZDusLiq3bduGnj17Ijg4GCdPnkS9evVw7tw5CCHQuHHj8siRiIiIiCo4q09/f/DBBxgzZgyOHDkCBwcH/Prrr7h48SJat26NF198sTxyJCIiIqIKzuqi8vjx4xgwYAAAQKVSITs7G5UqVcKnn36KL774oswTJCIiIqKKz+qi0snJCTk5OQAAf39/nDlzRp52/fr1ssuMiIiIiGyG1ddUtmjRAn/88Qfq1KmDbt26YcyYMfj777+xatUqtGjRojxyJCIiIqIKzuqicsaMGcjMzAQATJgwAZmZmVixYgWqV69u8U3SiYiIiOjxYnVRGRISIv/f0dER33//fZkmRERERES2x+prKkNCQnDjxg2zeHp6uknBSURERERPDquLynPnzkGv15vFc3Jy8M8//5RJUkRERERkWyw+/b127Vr5/5s3b4ZWq5Wf6/V6bNu2DUFBQWWaHBERERHZBouLyt69ewMAJEmS71NpZGdnh6CgIHz11VdlmhwRERER2QaLi0qDwQAACA4OxoEDB+Dp6VluSRERERGRbbF69HdycnJ55EFERERENszqgToAsHPnTvTo0QPVq1dHjRo10LNnT+zevbuscyMiIiIiG2F1UblkyRJ06NABjo6OGDlyJN544w1oNBq0b98ey5YtK48ciYiIiKiCk4QQwpoX1K5dG8OGDcNbb71lEp8xYwbmzp2L48ePl2mCtiQjIwNarRY6nQ4uLi6POh0iIqKKY5n0qDMoe/2tKqEemK3UF1YfqTx79ix69OhhFu/ZsyevtyQiIiJ6QlldVAYEBGDbtm1m8W3btiEgIKBMkiIiIiIi22Lx6O9Bgwbhm2++wZgxYzBy5EgkJSWhZcuWkCQJe/bsQXR0NL755pvyzJWIiIiIKiiLr6lUKpVISUmBt7c3Vq9eja+++kq+frJ27dp455130KtXr3JNtqKzlWseiIiIHjpeU/nAbKW+sPhIZeHa87nnnsNzzz1XLgkRERERke2x6ppKSXoMv2UQERERUalZ9Ys6NWvWvG9hefPmzVIlRERERES2x6qicuLEidBqteWVCxERERHZKKuKypdffhne3t7llQsRERER2SiLr6nk9ZREREREVBKLi0orf82RiIiIiJ4gFp/+NhgM5ZkHEREREdkwq3+mkYiIiIioKBaVRERERFRqLCqJiIiIqNRYVBIRERFRqbGoJCIiIqJSq9BF5YQJEyBJksnD19dXni6EwIQJE+Dv7w+NRoM2bdrg6NGjJvPIycnBm2++CU9PTzg5OaFnz564dOmSSZu0tDRERkZCq9VCq9UiMjIS6enpD6OLRERERI+FCl1UAkDdunWRkpIiP/7++2952pdffokZM2Zg1qxZOHDgAHx9fdGxY0fcunVLbjN69GisXr0aMTEx2LNnDzIzM9G9e3fo9Xq5Tf/+/ZGUlITY2FjExsYiKSkJkZGRD7WfRERERLbMqp9pfBRUKpXJ0U
```

Comment:
Yes, this is because the sample contains data from the first 100k events, which covers only 3 days for zkLend.

Reply:
Yes.